README.md

```diff
@@ -1 +1,21 @@
 # SB Engineering Guides
+The SB Engineering Guides provide a way to document processes and best practices that have established themselves over time or resulted from spikes and meetings.
+
+Rather than just capturing how we do things at SB, this shall repo also document why we do it that way in the form of discussions inside a pull request.
```

Comment:
```suggestion
Rather than just capturing how we do things at SB, this repo shall also document why we do it that way in the form of discussions inside a pull request.
```